runner/src/server/plugins/engine/pageControllers/MiniSummaryPageController.ts

```diff
@@ -0,0 +1,35 @@
+import { HapiRequest, HapiResponseToolkit } from "server/types";
+import { SummaryViewModel } from "../models";
+import { PageController } from "./PageController";
+
+/**
+ * MiniSummaryPageController is for mini summary pages
+ */
+export class MiniSummaryPageController extends PageController {
+  isMiniSummaryPageController = true;
```

Comment:
if `isMiniSummaryPageController` isn't used, you can remove it. It's only useful if you're going to use it for something like filtering.